pkg/configmap/testdata/bundles/etcd.0.9.2/manifests/etcdoperator.v0.9.2.clusterserviceversion.yaml

```diff
@@ -17,7 +17,7 @@ spec:
 
     ### Reading and writing to etcd
 
-    Communicate with etcd though its command line utility `etcdctl` or with the API using the automatically generated Kubernetes Service.
```

Comment:
Is there any cause for concern that these diverge from the [source](https://github.com/operator-framework/community-operators/blob/1591b5e18d47129ed14e83ac2f9832d6cb41bb40/community-operators/etcd/0.9.4/etcdoperator.v0.9.4.clusterserviceversion.yaml#L166)

Reply:
I don't think so, all just one-time copied test files.

Reply:
Also :whistles: .... the etcd content is duplicated all over this repo.

Reply:
Yeah, it's... messy.